src/main/java/picard/util/IntervalListTools.java

```diff
@@ -236,13 +261,22 @@
             " broken up at integer multiples of this value. Set to 0 to NOT break up intervals.", optional = true)
     public int BREAK_BANDS_AT_MULTIPLES_OF = 0;
 
-    @Argument(shortName = "M", doc = "Selects between various ways in which scattering of the interval-list can happen.")
-    public IntervalListScatterer.Mode SUBDIVISION_MODE = IntervalListScatterer.Mode.INTERVAL_SUBDIVISION;
+    @Argument(shortName = "M", doc = "Selects the way in which scattering of the interval-list will happen.")
```

Comment:
Not sure what the style guide is for picard docs is. Maybe `The mode used to scatter the interval list.` would be less awkward.